src/uniforms/EquirectHdrInfoUniform.js

```diff
@@ -2,15 +2,16 @@ import { DataTexture, FloatType, RedFormat, LinearFilter, DataUtils, HalfFloatTy
 
 function binarySearchFindClosestIndexOf( array, targetValue, offset = 0, count = array.length ) {
 
-	let lower = 0;
-	let upper = count - 1;
+	let lower = offset;
+	let upper = offset + count - 1;
+
 	while ( lower < upper ) {
 
-		const mid = ~ ~ ( 0.5 * upper + 0.5 * lower );
+		const mid = ( lower + upper ) >> 1;
```

Comment:
Can we add a comment here stating what this does? Basically truncating the average value of `lower` and `upper`.

Reply:
Done, I've added comments to it